uint32 x_center
uint32 y_center
uint32 pixel_width
uint32 pixel_height
float32 rotation_degrees
